assets/styles/style.css

```diff
@@ -0,0 +1,604 @@
+  display: flex;
+}
+
+.display-flex-inline {
+  display: inline-flex;
+}
+
+.display-grid {
+  display: grid;
+}
+
+.display-grid-inline {
+  display: inline-grid;
+}
+
+.border-none {
+  border: none;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  display: block !important;
+}
+
+p {
+  font-size: 20px;
+}
+
+/* colors */
+ul {
+  padding: 0;
+}
+
+.success {
+  color: #36b37e;
+}
+
+.neutrals {
+  color: #172b4d;
+}
+
+.navbar {
+  height: 48px !important;
+  display: flex;
+  flex-direction: row;
+  flex-basis: 100%;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  background-color: rgb(255 255 255 / 80%);
+  left: 0;
+}
+
+.site-title {
+  color: #172b4d;
+  text-decoration: none;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 600;
+  transition: 500ms ease;
+}
+
+.site-title:hover {
+  color: #36b37f;
+}
+
+.wrapper {
+  width: 100%;
+  padding: 0;
+  min-height: 702.12px;
+}
+
+/* section-1 */
+.welcome {
+  background-image: url("../images/header-illsutration-mobile.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  object-fit: cover;
+  min-height: 648px;
+  padding: 64px 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.welcome p {
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.greetings,
+.section-heading {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: bold;
+  margin: 5px !important;
+}
+
+.separator {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37f;
+  margin-bottom: 58px;
+}
+
+.separator-blank {
+  margin-bottom: 58px;
+}
+
+.social-links {
+  list-style-type: none !important;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.social-links li a {
+  font-size: 20px;
+  text-decoration: none;
+  padding: 8px;
+  color: #505f79;
+  transition: 500ms ease;
+}
+
+.social-links li a:hover {
+  color: #36b37f;
+}
+
+.m-logo:hover {
+  filter: invert(88%) sepia(7%) saturate(4102%) hue-rotate(125deg) brightness(71%) contrast(89%);
+}
+
+.footer-mobile {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: center;
+  padding: 44px 0 256px;
+}
+
+.footer-end {
+  display: flex;
+  flex-direction: column;
+  padding-top: 21px;
+  align-items: center;
+  height: 34px;
+}
+
+.footer-end-line {
+  width: 134px;
+  height: 5px;
+  border-radius: 10px;
+  background-color: #0d151e;
+}
+
+.down-arrow {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border-radius: 32px;
+  padding: 8px;
+  position: absolute;
+  border: 1px solid #dfe1e6;
+  top: 640px;
+  text-decoration: none !important;
+  color: #172b4d;
+  transition: 500ms ease;
+}
+
+.down-arrow:hover {
+  color: white;
+  background-color: #36b37f;
+}
+
+/* section recent-works */
+.recent-works {
+  padding: 64px 24px;
+  display: flex;
```

Comment:
[Optional] I recommend that you adjust the margin of the cards to correspond to the design by figma.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199685010-0e9dd8f6-c0e2-40b3-9471-2a2ee3c0f438.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199684417-b5c5a056-40e7-4a6c-91b8-33daa7135c14.png)